examples/websocket_client.py

```
@@ -0,0 +1,281 @@
+                # Start listening for messages
+                asyncio.create_task(self._listen_for_messages())
+                
+                return payload
+            else:
+                raise Exception(f"Failed to join channel: {response_data}")
+                
+        except Exception as e:
+            logger.error(f"❌ Connection failed: {e}")
+            raise
+    
+    async def _listen_for_messages(self):
```

Ruff (0.11.9)

Missing return type annotation for private function `_listen_for_messages`

Add return type annotation: `None`

(ANN202)